definitions/npm/mysql_v2.x.x/flow_v0.28.x-/mysql_v2.x.x.js

```diff
@@ -29,13 +29,15 @@ declare module 'mysql' {
     ssl?: string | Object
   };
 
-  declare type QueryOptions = {
-    sql: string,
-    typeCast?: boolean | (field: Object, next: Function) => any,
-    nestTables?: boolean | string, // string form is a separator used to produce column names
-    values?: Array<mixed>,
-    timeout?: number
-  } | string;
+  declare type QueryOptions =
+    | {
```

Comment:
Is this prettier doing its thing? If that's the case I'll revert this one change before merging the PR.

Reply:
Yeah, I guess so. This just ran whenever I tried to check in something in this repo. Apparently ran by a precommit hook of sorts?

Reply:
Yeah that happens :/ In that case this LGTM, I'll make that tweak and merge it in tonight.

Reply:
Thanks 👍

Reply:
Hey @GAntoine! Is there anything blocking this PR to be merged? Happy to help out with any questions.

Reply:
Btw, I also just snuck in this little addition while I was at it: 2523b515eb1adf49e2f6812a272fa333fa757be3

Reply:
Not exactly, I just need to get around to it, but things have been hectic the last few days. If you're able to fix the prettier stuff (tip: commit without running the post-commit hook by commenting it out in package.json) and fix the failing specs, I'll merge it right in.